src/chttpd_cors.erl

```diff
@@ -306,6 +316,35 @@ get_cors_config(#httpd{cors_config = undefined}) ->
 get_cors_config(#httpd{cors_config = Config}) ->
     Config.
 
+cors_config(Host, Key, Default) ->
+    config:get(cors_section(Host), Key,
+                     config:get("cors", Key, Default)).
+
+cors_section(Host0) ->
+    {Host, _Port} = split_host_port(Host0),
```

Comment:
Then I imagine the following should do the same.

```
{Scheme, HostPort, _, _, _} = mochiweb_util:urlsplit(Host),
lists:flatten(io_lib:format("cors:~s://~s", [Scheme, hd(string:tokens(HostPort, ":"))])).
```

But as I said, I'm fine with moving it here verbatim.